modules/llamacpp_hf.py

```diff
@@ -12,11 +12,20 @@
 
 if torch.cuda.is_available() and not torch.version.hip:
     try:
-        from llama_cpp_cuda import Llama
+        import llama_cpp_cuda
     except:
-        from llama_cpp import Llama
+        llama_cpp_cuda = None
+        import llama_cpp
```

Comment:
If you remove the indent from this line, then you can freely swap between llama_cpp_cuda and llama_cpp, which I believe is what you said that you wanted.
```python
    try:
        from llama_cpp_cuda import Llama
        import llama_cpp_cuda
    except:
        from llama_cpp import Llama
        llama_cpp_cuda = None
    import llama_cpp
```

Reply:
Oops, my bad, I missed that. It should be fixed now.